docs/Makefile

```
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = .
```

Some of the sklearn-contrib commands require a `PAPER` variable.

```suggestion
SOURCEDIR     = .
PAPER         = https://arxiv.org/abs/2402.01484
```